QA/conf/google_api_conf.py

```diff
@@ -0,0 +1,9 @@
+"""
+This file contains the configuration details to fetch events from google api
+"""
+email_id="test@qxf2.com"
+mintime="2025-07-10T09:00:00+05:30"
+maxtime="2025-07-10T17:00:00+05:30"
+date="07/10/2025"
+
+free_slots_for_10_july_2025=['09:00-09:30', '09:30-10:00', '10:00-10:30', '10:30-11:00', '11:00-11:30', '11:30-12:00', '13:00-13:30', '13:30-14:00', '14:00-14:30', '14:30-15:00', '16:00-16:30']
```

Comment:
Can you give the decent variable name ,I think you dont need to specify the date in the variable name

Reply:
I have changed the name of the variable